Sources/tinys3/responses/S3HeadResponse.swift

```diff
@@ -16,7 +16,7 @@ public struct S3HeadResponse {
         guard
             let contentLengthString = response.value(forHTTPHeaderField: .contentLength),
             let contentLength = Int(contentLengthString),
-            let eTag = response.value(forHTTPHeaderField: .eTag)?.replacingOccurrences(of: "\"", with: ""),
+            let eTag = response.value(forHTTPHeaderField: .eTag),
```

Comment:
As [per the HTTP spec/RFC](https://www.rfc-editor.org/rfc/rfc2616#section-3.11), the literal quotes are fully part of the ETag string (in fact, the quoted string might be prefixed with unquoted `W/`, e.g. `W/"a1b2c3"`, so it'd be invalid to remove quotes from such ETag).

```
      entity-tag = [ weak ] opaque-tag
      weak       = "W/"
      opaque-tag = quoted-string

      quoted-string  = ( <"> *(qdtext | quoted-pair ) <"> )
```

Keeping the quotes also helps keeping consistency, otherwise some parts of the code like here would remove the quotes from the ETag, while other parts of the code and the API itself (like when including an ETag into a request we send to AWS S3, e.g. `S3MultipartUploadCompleteXMLBuilder`) still expect to have those quotes in the ETags.

We wouldn't want fetching the `eTag` of an `S3HeadResponse` then later including that fetched `eTag` value to a subsequent request… to break the request just because we trimmed the quotes from the eTag during `S3HeadResponse`